tests/unit/amo/components/TestAddonReview.js

```diff
@@ -126,6 +126,29 @@ describe(__filename, () => {
       });
   });
 
+  it('a review without text feedback will show as no text in textarea and &nbsp; in review list display', () => {
```

Comment:
We generally try to name tests so they start with a verb, so the "it ..." makes sense, like `it('can...')` and `it('should...')`.